docs/HowToCertificatesRotation.md

```diff
@@ -0,0 +1,486 @@
+
+## Introduction
+
+When using TLS encryption, it is necessary to rotate key pair certificates before the certificates expire.
+To avoid downtime, NuoDB Admin Processes (APs) with the new certificates should be introduced in a rolling fashion.
+
+This document expands on [Security Model of NuoDB in Kubernetes](./HowtoTLS.md) and explains how to configure proper certificate rotation in NuoDB Kubernetes deployments.
+
+> **NOTE**: For non-Kubernetes deployments, see  [Enabling TLS encryption, Rotating TLS Key Pair Certificates](https://doc.nuodb.com/nuodb/latest/deployment-models/physical-or-vmware-environments-with-nuodb-admin/domain-operations/enabling-tls-encryption/rotating-key-pair-certificates/).
+This page expands on the product documentation and is specific to this Helm Chart repository.
+
+A high-level overview of the certificate rotation is provided in [About Key Rotation](#about-key-rotation) section.
+After getting familiar with the key concepts, step-by-step instructions are available in the [NuoDB TLS Key Rotation](#nuodb-tls-key-rotation) section.
+It is recommended to read through the entire page to get a common understanding of the key rotation process before following the steps.
+
+### Terminology
+
+- `Key` = a combination of a private key with its corresponding X509 certificate chain.
+These keys are usually saved in a PKCS12 file such as `nuoadmin.p12`.
+- `NuoDB Admin` = The administrative interface for domain and database management.
+For information on how to start the NuoDB Admin tier, see [Admin Chart](../stable/admin/README.md).
+- `CA` = Certificate Authority
+- `Sever Key` = The key pair certificate used by NuoDB Admin.
+
+## About Key Rotation
+
+NuoDB supports various key models for TLS keys used by the `NuoDB Admin` and database processes.
+
+> **NOTE**: For information on how to configure different TLS key models, see [Security Model of NuoDB in Kubernetes](./HowtoTLS.md).
+
+The [Shared Admin Key + Intermediate CA](./docs/HowtoTLS.md#intermediate-ca) model is used by default in NuoDB Kubernetes deployments.
+This means that the same key pair certificate is used by all APs, a Certificate Authority (CA) is used to sign them and the CA certificate is trusted by all processes in the domain.
+
+If a CA certificate needs to be renewed, then both the keystore and the truststore must be updated for all APs, and by extension, all database processes.
+As stated, processes with the new certificates should be introduced in a rolling fashion to avoid downtime, which means that processes with the old key pair certificate can verify processes with the new key pair certificate.
+Since the new key pair certificates must be verified using a new trusted certificate, existing processes (admin and database) must have the new trusted certificate propagated to their truststores before the new key pair certificates can be introduced.
+
+To simplify updating and propagating new trusted certificates, run `nuocmd add trusted-certificate`.
```

Comment:
link to https://doc.nuodb.com/nuodb/latest/reference-information/command-line-tools/nuodb-command/nuocmd-reference/#add-trusted-certificate?